src/Illuminate/Auth/Access/Gate.php

```diff
@@ -705,6 +708,10 @@ protected function callPolicyMethod($policy, $method, $user, array $arguments)
         if ($this->canBeCalledWithUser($user, $policy, $method)) {
             return $policy->{$method}($user, ...$arguments);
         }
+
+        if (! $user) {
+            throw new \Illuminate\Auth\AuthenticationException();
```

Comment:
Please import this (but don't update the phpdoc).

Reply:
done